src/components/CoinsList/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
I still recommend naming the component files with their descriptive name, e.g.` CoinsList.tsx` or` CoinsList.component.tsx`.
Who do you choose to keep their names just `index.tsx`?